demos/matrix_free/opts.benard.fs.txt

```diff
@@ -0,0 +1,25 @@
+-mat_type matfree
+-Ra 200.0
+-Pr 6.8
+-snes_monitor
+-ksp_type gmres
+-ksp_gmres_restart 30
+-ksp_gmres_modifiedgramschmidt
+-ksp_monitor_true_residual
+-pc_type fieldsplit
+-pc_fieldsplit_type multiplicative
+# Split Navier-Stokes from temperature
+-pc_fieldsplit_0_fields 0,1
+-pc_fieldsplit_1_fields 2
+# Assemble Navier-Stokes and hit it with mumps
+-fieldsplit_0_ksp_type preonly
+-fieldsplit_0_pc_type python
+-fieldsplit_0_pc_python_type firedrake.AssembledPC
+-fieldsplit_0_assembled_pc_type lu
+-fieldsplit_0_assembled_pc_factor_mat_solver_package mumps
+# Assemble and factor temperature convection-diffusion
+-fieldsplit_1_ksp_type preonly
+-fieldsplit_1_pc_type python
+-fieldsplit_1_pc_python_type firedrake.AssembledPC
+-fieldsplit_1_assembled_pc_type lu
+
```

Comment:
GOt it.


On Fri, Aug 12, 2016 at 3:34 AM, Miklós Homolya <notifications@github.com>
wrote:

> In demos/matrix_free/opts.benard.fs.txt
> <https://github.com/firedrakeproject/firedrake/pull/853#discussion_r74557339>
> :
>
> > +-pc_fieldsplit_type multiplicative
> > +# Split Navier-Stokes from temperature
> > +-pc_fieldsplit_0_fields 0,1
> > +-pc_fieldsplit_1_fields 2
> > +# Assemble Navier-Stokes and hit it with mumps
> > +-fieldsplit_0_ksp_type preonly
> > +-fieldsplit_0_pc_type python
> > +-fieldsplit_0_pc_python_type firedrake.AssembledPC
> > +-fieldsplit_0_assembled_pc_type lu
> > +-fieldsplit_0_assembled_pc_factor_mat_solver_package mumps
> > +# Assemble and factor temperature convection-diffusion
> > +-fieldsplit_1_ksp_type preonly
> > +-fieldsplit_1_pc_type python
> > +-fieldsplit_1_pc_python_type firedrake.AssembledPC
> > +-fieldsplit_1_assembled_pc_type lu
> > +
>
> Blank line at the bottom.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/firedrakeproject/firedrake/pull/853/files/a1749eaed823ed1ce3c110187f3b43e4f8a24669#r74557339>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADl6L2R3a_1-WFkC_kHUqE7V92CzcMX1ks5qfDALgaJpZM4JiV-g>
> .
>